Plugin/CppApi/ArcGISRuntimeToolkit.pro

```diff
@@ -62,17 +62,15 @@ CONFIG(release, debug|release) {
   BUILDTYPE = debug
 }
 
-!android:!win32: PLATFORM_OUTPUT = $$PLATFORM
-android: PLATFORM_OUTPUT = $$PLATFORM/$$ANDROID_ARCH
 win32: {
   contains(QMAKE_TARGET.arch, x86_64): {
     PLATFORM_OUTPUT = $$PLATFORM/x64
   } else {
     PLATFORM_OUTPUT = $$PLATFORM/x86
   }
 }
+else: {
+    PLATFORM_OUTPUT = $$PLATFORM
+}
 
 DESTDIR = $$PWD/output/$$PLATFORM_OUTPUT
-OBJECTS_DIR = $$DESTDIR/$$BUILDTYPE/obj
-MOC_DIR = $$DESTDIR/$$BUILDTYPE/moc
-RCC_DIR = $$DESTDIR/$$BUILDTYPE/qrc
```

Comment:
Why remove these?

Reply:
With these lines, the object/moc/rcc files for the different android architectures overwrite each other and the linking fails.

I'm not sure why we every tried to enforce in-source builds in the first place. Aside from being bad practice, the default location for build files seems perfectly servicible.